dali/kernels/imgproc/flip_cpu.h

```diff
@@ -23,6 +23,9 @@
 
 namespace dali {
 namespace kernels {
+
+constexpr int sample_ndim = 5;
```

Comment:
can you use this in all the places where there is a 5 hardcoded?

Reply:
done